stats/weapons.json

```diff
@@ -2194,6 +2194,43 @@
 		"weaponWav": "mgbar3.ogg",
 		"weight": 400
 	},
+	"MG2-Tower": {
+		"buildPoints": 125,
+		"buildPower": 25,
+		"damage": 14,
+		"designable": 1,
+		"effectSize": 100,
+		"facePlayer": 1,
+		"firePause": 6,
+		"flags": "ShootAir",
+		"flightGfx": "fxtracer_twin.pie",
+		"flightSpeed": 1000,
+		"hitGfx": "FXMETHIT.PIE",
+		"hitpoints": 125,
+		"id": "MG2-Tower",
+		"longHit": 60,
+		"longRange": 768,
```

Comment:
Ranges might need to be looked at again since the regular MG tower has 830 longrange. This weapon shares the same range as the tank Twin MG so maybe a copy-paste error?

Reply:
Yup, thank you. Fixed